www-apps/gitea/gitea-1.17.0.ebuild

```diff
@@ -0,0 +1,125 @@
+# Copyright 2016-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit fcaps go-module tmpfiles systemd flag-o-matic
+
+DESCRIPTION="A painless self-hosted Git service"
+HOMEPAGE="https://gitea.io https://github.com/go-gitea/gitea"
+
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/go-gitea/gitea.git"
+else
+	SRC_URI="https://github.com/go-gitea/gitea/releases/download/v${PV}/gitea-src-${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+fi
+
+S="${WORKDIR}/${PN}-src-${PV}"
+
+LICENSE="Apache-2.0 BSD BSD-2 ISC MIT MPL-2.0"
+SLOT="0"
+IUSE="+acct pam sqlite pie"
+
+DEPEND="
+	acct? (
+		acct-group/git
+		acct-user/git[gitea] )
+	pam? ( sys-libs/pam )"
+RDEPEND="${DEPEND}
+	dev-vcs/git"
+
+DOCS=(
+	custom/conf/app.example.ini CONTRIBUTING.md README.md
+)
+FILECAPS=(
+	-m 711 cap_net_bind_service+ep usr/bin/gitea
+)
+
+RESTRICT="test"
```

Comment:
Why? It seems like tests depend on nodejs, and with that installed the vast majority of tests seem to work for me.